raiden/lightclient/handlers/light_client_message_handler.py

```diff
@@ -54,15 +54,18 @@ def store_light_client_protocol_message(cls, identifier: int, message: Message,
         )
 
     @classmethod
-    def update_stored_msg_set_signed_data(
+    def update_offchain_light_client_protocol_message_set_signed_message(
```

Comment:
i don't think this name makes too much sense. it's 9 (!) words long and redundant.

to start with we have `update offchain light client protocol message`:
- the word `protocol` doesn't really add much (`light client message` would be just as clear).
- i've read @marcosmartinez7's comment, but i don't think it's necessary to say whether this is off-chain or on-chain.

and then `set signed message`: we already knew this was an update, but now we are also saying this writes a specific field. so why say _update_ in the first place?

i would've named this `set_light_client_message_signed_message` or `set_signed_message_for_light_client_message`.

Reply:
I mean if we are going to be strict on this well, `set_light_client_message_signed_message` is not good neither, since we are inside the LightClientMessageHandler then we should just call this method `set_signed_data` so the call could be `LightClientMessageHandler.set_signed_data`  but i don't know i can change it again, i agree that the new name is really large but we need to stablish a criteria on this or each time i will need to change it several times.

Reply:
it's fine, leave it like this :+1:  

just keep this in mind for next time and i suggest we're just bold and change it to something like `set_signed_data` directly. 

your suggestion is actually ideal :clap:

Reply:
Im ok with the two proposals, just wanted to differenciate on the signature offchain from onchain.

Reply:
let's be bold :)

Reply:
Ok as we spoke we have an issue already to rename and refactor the class LightClientMessageHandler so we are going to keep things like this for now and make all the refactor on that issue.